include/swift/AST/DiagnosticsSema.def

```diff
@@ -5786,6 +5786,9 @@ ERROR(distributed_property_cannot_be_static,none,
 ERROR(distributed_property_can_only_be_computed,none,
       "%kind0 cannot be 'distributed', only computed properties can",
       (const ValueDecl *))
+ERROR(distributed_property_accessor_only_get_can_be_distributed,none,
+      "only getters may be 'distributed'",
+      ())
```

Comment:
This seems to be unused.

Reply:
done. [4ff3b9c](https://github.com/swiftlang/swift/pull/74462/commits/4ff3b9c8f03fecfa36f15ef2e436f3ad10697b70)